### PR TITLE
[UI] fix: update record height if query change

### PR DESCRIPTION
**Pull request by leiyre**
When query changes, the height of the record must be recalculated for the record to display scroll or not.

### CODE REVIEW

**Comment by frascuchon** [comment]
This behaviour only affect to text-classification tasks?

**Comment by leiyre** [comment]
> This behaviour only affect to text-classification tasks?

I have applied the behaviour also for token classifier

File: frontend/components/text-classifier/RecordString.vue [hunk 1]
```diff
@@ -57,17 +57,22 @@ export default {
...
-      this.scrollHeight = this.$refs.list.clientHeight + padding;
-    }
+    this.getRecordHeight();
   },
-
   methods: {
     isList(record) {
       return Array.isArray(record);
     },
+    getRecordHeight() {
```
Comment (frascuchon) [inline frontend/components/text-classifier/RecordString.vue]:
```suggestion
    calculateScrollHeight() {
```

File: frontend/components/text-classifier/RecordString.vue [hunk 2]
```diff
@@ -57,17 +57,22 @@ export default {
...
   }),
+  updated() {
+    this.getRecordHeight();
+  },
   mounted() {
-    if (this.$refs.list) {
-      const padding = 2;
-      this.scrollHeight = this.$refs.list.clientHeight + padding;
-    }
+    this.getRecordHeight();
```
Comment (frascuchon) [inline frontend/components/text-classifier/RecordString.vue]:
```suggestion
    this.calculateScrollHeight();
```

File: frontend/components/text-classifier/RecordString.vue [hunk 3]
```diff
@@ -57,17 +57,22 @@ export default {
     allowScroll: false,
     scrollHeight: undefined,
   }),
+  updated() {
+    this.getRecordHeight();
```
Comment (frascuchon) [inline frontend/components/text-classifier/RecordString.vue]:
```suggestion
    this.calculateScrollHeight();
```

File: frontend/components/token-classifier/RecordTokenClassification.vue [hunk 4]
```diff
@@ -121,6 +148,12 @@ export default {
       return this.dataset.viewSettings.annotationEnabled;
     },
   },
+  updated() {
+    this.getRecordHeight();
```
Comment (frascuchon) [inline frontend/components/token-classifier/RecordTokenClassification.vue]:
```suggestion
    this.calculateScrollHeight();
```

File: frontend/components/token-classifier/RecordTokenClassification.vue [hunk 5]
```diff
@@ -121,6 +148,12 @@ export default {
...
+  },
+  mounted() {
+    this.getRecordHeight();
```
Comment (frascuchon) [inline frontend/components/token-classifier/RecordTokenClassification.vue]:
```suggestion
    this.calculateScrollHeight();
```

File: frontend/components/token-classifier/RecordTokenClassification.vue [hunk 6]
```diff
@@ -198,6 +231,12 @@ export default {
       entities.splice(found, 1);
       this.updateRecordEntities(entities);
     },
+    getRecordHeight() {
```
Comment (frascuchon) [inline frontend/components/token-classifier/RecordTokenClassification.vue]:
```suggestion
    calculateScrollHeight() {
```

**Pull request merged**